packages/foam-vscode/package.json

```diff
@@ -723,6 +723,11 @@
           "type": "boolean",
           "default": false,
           "description": "Whether to open the graph on startup."
+        },
+        "foam.graph.nodeFontSizeMultiplier": {
```

Comment:
mmmm..  I am not sure about exposing something like this a configuration, can you talk me through it?